Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml

```diff
@@ -267,25 +266,24 @@
                             </VisualStateManager.VisualStateGroups>
                         </Grid>
 
-                        <Border Grid.Row="1"
-                                HorizontalAlignment="Stretch"
-                                Background="{TemplateBinding ContentBackground}" />
-                        <ContentControl x:Name="PART_RibbonContent"
-                                        Grid.Row="1"
-                                        HorizontalAlignment="Stretch"
-                                        HorizontalContentAlignment="Stretch"
-                                        Background="{TemplateBinding ContentBackground}">
-                            <ContentControl.Resources>
-                                <Storyboard x:Name="TabChangedStoryboard">
-                                    <PopInThemeAnimation TargetName="PART_RibbonContent" />
-                                </Storyboard>
-                                <Style TargetType="AppBarElementContainer">
-                                    <Setter Property="VerticalAlignment" Value="Stretch" />
-                                    <Setter Property="VerticalContentAlignment" Value="Center" />
-                                    <Setter Property="Margin" Value="0,0,1,0" />
-                                </Style>
-                            </ContentControl.Resources>
-                        </ContentControl>
+
+                        <Border x:Name="PART_RibbonContentBorder"
+                                Background="{TemplateBinding ItemBackground}" Grid.Row="1"
```

Comment:
```cs
SelectionChanged += (NavigationView sender, NavigationViewSelectionChangedEventArgs args) =>
            {
                _ribbonContentBorder.Background = (sender.SelectedItem as Control).Background;
```
I'm a bit confused, isn't this the thing you're setting above in the SelectionChanged event? So then what's the point of using the binding and exposing the ItemBackground property?

Reply:
Thanks for pointing that out, I forgot to remove the binding from when I was debugging it.